Packages/MIES/MIES_SweepFormula_PSX.ipf

```diff
@@ -329,60 +358,67 @@ static Function [WAVE sweepDataOff, variable offset] PSX_OffsetSweepData(WAVE sw
 	return [output, offset]
 End
 
+static Function PSX_AddDefaultFilterFrequencies(WAVE filter, variable riseTau, variable decayTau, variable filterType)
+
+	variable fac
+
+	switch(filterType)
+		case PSX_FILTER_SWEEP:
+			fac = 3
+			break
+		case PSX_FILTER_DECONV:
+			fac = 4
+			break
+		default:
+			ASSERT(0, "Invalid case")
+	endswitch
+
+	filter[%$"Filter Low (Default)"]  = 1 / (fac * pi * riseTau)
```

Comment:
@t-b if rise and decay tau are in ms, they should be divided by 1000

Reply:
Good point. Until know they were always used relative to the input wave x deltas, so their real unit was not documented. I've changed that.

And also added a conversion factor. In MIES we always prefer ` * MILLI_TO_ONE` as that is less likely to be used wrongly.